charts/opensearch/values.yaml

```diff
@@ -104,6 +104,10 @@ config:
 extraEnvs: []
 #  - name: MY_ENVIRONMENT_VAR
 #    value: the_value_goes_here
+# 2.12 onwards a custom strong password needs to be provided in order to setup demo admin user.
```

Comment:
This should be clearly state as chartVersion 2.18.0 and the appVersion 2.12.0 onward, or it will confuse people on this.